uint16 gripper_width
uint16 status
uint16 max_width
uint16 min_width